e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -1,290 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Tests here were split into 2 specs for ease of maintenance and make the test faster by preventing unnecessary URL visit to a channel that cause prolong page reload every time.